benchmarks/workloads.py

```diff
@@ -25,7 +25,8 @@ class WorkloadBuilder:
     @classmethod
     def wrap_module(cls, ops: list[str]) -> str:
         """Wrap a list of operations as a module."""
-        workload = f'"builtin.module"() ({{\n  {"\n  ".join(ops)}\n}}) : () -> ()'
+        body = "\n  ".join(ops)
```

Comment:
it wasn't happy about escape sequences inside the {} below, this is only supported in Python 3.12 and up.